### PR TITLE
Fix: Add additional trigger in BHT + fix display

### LINKED ISSUE #11: Better instructions for the resting state task

### PR DESCRIPTION
**Pull request by esavary**
- Add triggers for breath out during BHT. 
- - Add triggers for fixation point during RS. 
- Improve introduction text in RS. closes #11 
- Fix the size of the fixation points in calibration
-  Fix server address

### CODE REVIEW

**Comment by esavary** [comment]
The task durations are currently hardcoded in the write_event_file.py script. These values are also hardcoded in the Python files generated by Psychopy. If we continue using the .psyexp files, it might be a more straightforward and cleaner approach to extract and parse the task durations directly from there instead of from the Python files. What are your thoughts on this @oesteban ?

File: task-rest_bold.psyexp [hunk 1]
```diff
@@ -153,36 +140,108 @@
         <Param val="" valType="extendedCode" updates="constant" name="Before JS Experiment"/>
         <Param val="" valType="extendedCode" updates="constant" name="Begin Experiment"/>
         <Param val="" valType="extendedCode" updates="constant" name="Begin JS Experiment"/>
-        <Param val="ioServer.getDevice(&quot;tracker&quot;).sendMessage(&quot;hello rs&quot;);&amp;#10;" valType="extendedCode" updates="constant" name="Begin JS Routine"/>
-        <Param val="ioServer.getDevice('tracker').sendMessage(&quot;hello rs&quot;)&amp;#10;" valType="extendedCode" updates="constant" name="Begin Routine"/>
+        <Param val="ioServer.getDevice(&quot;tracker&quot;).sendMessage(&quot;start movie&quot;);&amp;#10;" valType="extendedCode" updates="constant" name="Begin JS Routine"/>
```
Comment (oesteban) [inline task-rest_bold.psyexp:143]: Does this respond to some advance on theaxonlab/hcph-sops#74 ?

File: task-bht_bold.psyexp [hunk 2]
```diff
@@ -68,18 +68,18 @@
...
     <Routine name="breath_out">
-      <PolygonComponent name="polygon_2" plugin="None">
+      <PolygonComponent name="polygon1" plugin="None">
         <Param val="center" valType="str" updates="constant" name="anchor"/>
         <Param val="rgb" valType="str" updates="constant" name="colorSpace"/>
         <Param val="1" valType="num" updates="constant" name="contrast"/>
         <Param val="False" valType="bool" updates="None" name="disabled"/>
         <Param val="" valType="code" updates="None" name="durationEstim"/>
-        <Param val="1.0000, 0.2941, -1.0000" valType="color" updates="constant" name="fillColor"/>
+        <Param val="1.0000, 1.0000, -1.0000" valType="color" updates="constant" name="fillColor"/>
```
Comment (oesteban) [inline task-bht_bold.psyexp:77]: why this color change?

**Review by oesteban — COMMENTED** [review]
A couple of questions before merging, but it looks okay :)

File: task-rest_bold.psyexp (hunk 1 above)
Comment (esavary) [reply task-rest_bold.psyexp:143]: Yes, but also connected to https://github.com/TheAxonLab/hcph-sops/issues/74#91 as I have implemented the event file writing for resting state within this context.

File: task-bht_bold.psyexp (hunk 2 above)
Comment (esavary) [reply task-bht_bold.psyexp:77]: To differentiate more with the last breath (before it was orange, now it is yellow.)

**Comment by esavary** [comment]
No problem, but are you sure about the color choice? The distinction from the orange used for the last breath was subtle, and the green used for the last breath isn't very different either.

**Comment by esavary** [comment]
Hi! Thanks for the review,  can I go ahead and merge it?

**Comment by oesteban** [comment]
Sure! Go ahead

**Pull request merged**